actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -589,10 +589,16 @@ def link_to_remote_options?(options)
         end
 
         def add_method_to_attributes!(html_options, method)
-          if method && method.to_s.downcase != "get".freeze && html_options["rel".freeze] !~ /nofollow/
-            html_options["rel".freeze] = "#{html_options["rel".freeze]} nofollow".lstrip
+          @_method_downcase_cache ||= {}
```

Comment:
Since this is in a private method, hopefully this doesn't have the same security concern that the dasherize change had? https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/url_helper.rb#L577

Reply:
It is still a problem. `link_to :foo, method: params[:method]`. I don't think caching calculation using instance variables is a good idea, even more in a case where the micro benchmarks fall within error.

Reply:
Do you have a preference on this? Maybe a constant or something of the common use cases? It looked like this was doing `:get` => `'get'` for each link on the page.

Reply:
Does it show up on a larger-than-micro benchmark?

Reply:
Does `/\Aget\z/i.match?(method)` offer any improvement?

Reply:
Just double check this and it was actually `delete` that I was thinking of. This code gets triggered when you pass in a `method` argument to `link_to`. In my use case, it's 1000s of links that have `method: :delete`.

Could add a constant for the standard REST verbs here?

Reply:
For reference - with `memory_profiler` and separating out the other change, on a page with 12880 delete links this reduced allocated memory by 1.54MB of string allocations at a cost of a 7.8KB increase in retained memory